sdk/src/client/state.ts

```diff
@@ -19,14 +18,19 @@ export async function _prepareAndSignInitialState(
     params: CreateChannelParams
 ): Promise<{ channel: Channel; initialState: State; channelId: ChannelId }> {
     const { initialAllocationAmounts, stateData } = params;
+
+    if (!stateData) {
+        throw new Errors.MissingParameterError("State data is required for creating the channel.");
+    }
```

Comment:
As we have discussed, `stateData` can be empty (`0x`) for some Adjudicators, therefore it might make sense to have an `Adjudicator` class with methods to validate state defined.
Nevertheless, this validation is definitely done on the BE, so I am not sure what's the best approach here.

Reply:
We can see if it will be needed, and we can add it at any moment